hs/test-examples/features/use_strict.txt

```diff
@@ -0,0 +1,7 @@
+Verifying knowledge assertions
+Verifying for generic connector
+  Verifying when ALL participants are honest
+  Verifying when NO participants are honest
+  Verifying when ONLY "A" is honest
+Checked 5 theorems; No failures!
+WARNING: unused variable: x at ./use_strict.rsh:11:15:id
```

Comment:
Woooaaaaah! But it *is* used! What's going on here?

Reply:
This example is a bit contrived since I think `use strict` will almost always be used at the top level. But, it errors due to the scope sensitivity of `use strict`. `x` is published outside of the scope `use strict` was declared in (`A.only`)

Reply:
I think that `'use strict'` is about monitoring the variables, not about monitoring the uses.